Phase01_Full-Text Search/src/main/java/ir/shelmos_search/language/LanguageProcessor.java

```diff
@@ -0,0 +1,47 @@
+package ir.shelmos_search.language;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+
+public class LanguageProcessor {
+    @Getter
+    @Setter
+    private Normalizer normalizer;
+    @Getter
+    @Setter
+    private Tokenizer tokenizer;
+    @Getter
+    @Setter
+    private String[] stopWords;
```

Comment:
You can use `@Getter` and `@Setter` one time by putting them above the class name.